assets/_scss/elements/_list.scss

```diff
@@ -1,29 +1,35 @@
-// Make lists unstyled by default
 ul, ol {
```

Comment:
I don't feel strongly either way about this. I wonder if it's easier to have the same rule applied to all selectors.